vernac/vernacentries.ml

```diff
@@ -886,6 +889,21 @@ let vernac_end_section {CAst.loc} =
 
 let vernac_name_sec_hyp {v=id} set = Proof_using.name_set id set
 
+let warn_proof_without_using_in_section =
+  CWarnings.create ~name:"proof-without-using-in-section" ~category:"vernacular"
+    (fun () -> strbrk "You should use the “Proof using [...].” syntax instead of “Proof.” to enable skipping this proof which is located inside a section. Give as argument to “Proof using” the list of section variables that are not needed to typecheck the statement but that are required by the proof.")
```

Comment:
This line is too long, please split it.

Reply:
Throughout the file, it seems to be the convention that there is no line break in the middle of an error message formatted using str or strbrk. Please suggest the  formatting of your choice, because I wouldn't know what to go for to improve the situation.

Reply:
Arf, we are not very good with coding conventions, sorry about that. I usually find long lines a pain to edit, so I'd do:

```ocaml
strbrk "An 80-character line where you start " ++
strbrk "the error message and then you keep " ++
strbrk "going."
```

But that can also be a pain to edit, so maybe you should just ignore this remark ;)